wiki/Registration/en.md

```diff
@@ -7,9 +7,11 @@
    ![The in-game registration form](img/ingame-registration.jpg "The registration form")
 
 2. Fill out all of the fields in the form.
+
```

Comment:
There shouldn't be any empty lines between the list items. I think lines 6 and 8 need to be removed instead.